assets/src/edit-story/components/library/panes/media/media3p/stories/categoriesPanel.js

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import styled from 'styled-components';
+
+/**
+ * Internal dependencies
+ */
+import CategoryPill from '../categoryPill';
+
+export default {
+  title: 'Stories Editor/Components/Media Categories',
+  component: CategoryPill,
+};
+
+const Container = styled.div`
+  background-color: ${({ theme }) => theme.colors.bg.v3};
+  padding: 1em;
+`;
+
+export const _default = () => {
+  return (
+    <Container>
+      <CategoryPill title="Unselected" />
+      <CategoryPill title="Selected" isSelected={true} />
```

Comment:
could use knobs or a simple onClick handler to allow toggling `isSelected` in the storybook

Reply:
Done. Added onClick handler in stories/categoryPIll.js